src/Console/GenerateSwaggerDocCommand.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace Mtrajano\LaravelSwagger\Console;
+
+use Illuminate\Console\Command;
+use Mtrajano\LaravelSwagger\FormatterManager;
+use Mtrajano\LaravelSwagger\Generator;
+use Mtrajano\LaravelSwagger\SwaggerDocsManager;
+
+class GenerateSwaggerDocCommand extends Command
+{
+    /**
+     * @var SwaggerDocsManager
+     */
+    private $swaggerDocsManager;
+
+    public function __construct(SwaggerDocsManager $swaggerDocsManager)
+    {
+        parent::__construct();
+
+        $this->swaggerDocsManager = $swaggerDocsManager;
+    }
+
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'laravel-swagger:generate
+                            {--format=json : The format of the output, current options are json and yaml}
+                            {--all-versions : Generate swagger docs for all versions}
+                            {--api-version= : The version of the swagger docs. Uses defaultVersion by default}';
```

Comment:
Also saw you removed the `--filter` option which is still being used in generator. That was added in case the user wanted to only generate swagger definitions for routes prefixed by `/api` etc..

Reply:
About `--api-version=*` seems a better idea.  About to use `defaultVersion` or just get the last version default I was thinking... 🤔Maybe the default behavior should be generate for all versions! It'll prevents the user change an older version and keep with out to date docs...

About `--filter` param: We think to use the `basePath` from each version config to filter the routes... Therefore, there is no need for the filter parameter.

what about it?